modules/axconfig/src/platform/pc-x86.toml

```diff
@@ -21,6 +21,7 @@ mmio-regions = [
     ["0xfec0_0000", "0x1000"],      # IO APIC
     ["0xfed0_0000", "0x1000"],      # HPET
     ["0xfee0_0000", "0x1000"],      # Local APIC
+    ["0xe_0000", "0x2_0000"],       #ACPI
```

Comment:
Where does this address come from?

Reply:
This address area is one of the places where RSDP can exist.
The `acpi` crate may access it to search for RSDP.

> The RSDP is either located within the first 1 KB of the [EBDA](https://wiki.osdev.org/EBDA) (Extended [BIOS](https://wiki.osdev.org/BIOS) Data Area) (a 2 byte real mode segment pointer to it is located at 0x40E), or in the memory region from 0x000E0000 to 0x000FFFFF (the main BIOS area below 1 MB). [Link](https://wiki.osdev.org/RSDP)